zerver/lib/email_mirror.py

```diff
@@ -126,7 +126,9 @@ def send_to_missed_message_address(address, message):
     # Testing with basestring so we don't depend on the list return type from
     # get_display_recipient
     if not isinstance(display_recipient, six.string_types):
-        display_recipient = ','.join([user['email'] for user in display_recipient])
+        drecipient = ','.join([user['email'] for user in display_recipient])
+    else:
+        drecipient = display_recipient
```

Comment:
I think `recipient_str` would be a better name for this variable, since it's basically a marshalled recipient object.